### ISSUE TITLE
App crashes when I click 'Scan lndconnect config' button - Calyxos on Pixel 5

### ISSUE THREAD

**Issue by Dbar139** [ZeusLN/zeus#555]
Installed the latest version of the app (0.5.2)  on my Pixel 5 running Calyxos v2.8. When clicking on 'Scan lndconnect config' the app crashes and returns me to the home screen of my device.

1. Launch Zeus app
2. Click on 'Scan lndconnect config'

**Expected behavior**
Was expecting camera app to launch, or a permission request to access the camera.  

 - Device: Google Pixel 5
 - OS: Calyxos 2.8
 - Zeus Version: v0.5.2 from F-Droid.

**Additional context**
I tried checking permissions and Zeus does have access to "Camera" and "Files and Media". 
Using microG Services.

**Comment by kaloudis** [comment]
It might be an issue with what you have on your clipboard - try clearing that and trying again

**Comment by Dbar139** [comment]
Thanks for the quick response. I tried clearing the clipboard but that didn't work for me unfortunately. The app still crashes when I click the scan button. I tried installing the app on a different phone Google Pixel 2 XL running stock Android 11 and had the exact same issue.

**Comment by kaloudis** [comment]
Can you try the v0.6.0-alpha1 release? It's on the release pages here on GitHub

**Comment by Dbar139** [comment]
The 0.6.0-alpha1 release solved the issue on my Pixel 2 XL. :)

For some reason I couldn't install the same apk on my Pixel 5 running Calyxos, it would not launch with an installer even though I have developer mode/install from unknown sources enabled. So I installed a package manager from F-Droid 'App Manager' which has an interceptor. I noticed when clicking on 'Scan lndconnect config' in v0.5.2 the interceptor shows that it attempts to access a service com.google.zxing which maybe is not part of the Calyxos build. So I installed 'Barcode Scanner' app and now it works perfectly!
